packages/Webkul/RMA/src/Listeners/Product.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Webkul\RMA\Listeners;
+
+use Illuminate\Foundation\Validation\ValidatesRequests;
+use Webkul\Product\Repositories\ProductRepository;
+use Webkul\Product\Contracts\Product as ProductContract;
+
+class Product
```

Comment:
This listner should be removed completely when columns are handled via attribute.